### PR TITLE
feat: (sfui2-534) slider selected element

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue

Closes [#534](https://vsf.atlassian.net/browse/SFUI2-534)

# Scope of work

Added centering the Slider on the inner element which gets focus (in horizontal and vertical modes) when moving by Tab key or on mouseclick. 

# Screenshots of visual changes

slider

# Checklist

- [ ] Self code-reviewed
- [ ] Changes documented
- [ ] Semantic HTML
- [ ] SSR-friendly
- [ ] Caching friendly
- [ ] a11y for WCAG 2.0 AA
- [ ] examples created
- [ ] removed old mitosis/examplesOld files
- [ ] created cypress tests

### CODE REVIEW

**Review by dkacper — APPROVED** [review]
I assume we provide onFocus scrolling to current item, but only if a user provides `tabIndex` to slider's items, right?

File: packages/sfui/frameworks/vue/components/VsfSlider/VsfSlider.vue [hunk 1]
```diff
@@ -64,6 +64,11 @@ const [container, slider] = useSlider(
 );
 const onClickPrev = () => slider.value?.prev();
 const onClickNext = () => slider.value?.next();
+
+const focusHandler = (event: FocusEvent) => {
+  const elementIndex = Array.prototype.indexOf.call(slider.value?.container.children, event.target);
```
Comment (Szymon-dziewonski) [inline packages/sfui/frameworks/vue/components/VsfSlider/VsfSlider.vue:69]:
```suggestion
  const elementIndex = Array.from(slider.value?.container.children).findIndex(el => el === event.target);
```
I think using `Array.from` and `findIndex` is more readable IMHO, what do you think?

File: packages/sfui/shared/slider/slider.ts [hunk 2]
```diff
@@ -71,9 +71,11 @@ export default class VSFSlider {
       const rect = children[index].getBoundingClientRect();
       const left = rect.left - container.getBoundingClientRect().left;
       const top = rect.top - container.getBoundingClientRect().top;
+      const centerHorizontal = container.clientWidth / 2 - rect.width / 2;
+      const centerVertical = container.clientHeight / 2 - rect.width / 2;
```
Comment (Szymon-dziewonski) [inline packages/sfui/shared/slider/slider.ts:75]:
```suggestion
      const centerVertical = container.clientHeight / 2 - rect.height / 2;
```
Should be height here?

File: packages/sfui/frameworks/vue/components/VsfSlider/VsfSlider.vue (hunk 1 above)
Comment (AdamPawlinski) [reply packages/sfui/frameworks/vue/components/VsfSlider/VsfSlider.vue]: Yes, that's better, thx.

File: packages/sfui/shared/slider/slider.ts (hunk 2 above)
Comment (AdamPawlinski) [reply packages/sfui/shared/slider/slider.ts]: Ok, changed it.

**Review by aniamusial — APPROVED** [review]

**Review by justyna-13 — APPROVED** [review]

**Review by Szymon-dziewonski — APPROVED** [review]
👍

**Pull request merged**